templates/zerver/help/emoji.md

```diff
@@ -0,0 +1,27 @@
+# Emoji
+Emoji allow you to add a little character and some fun to your messages! This guide shows you how to send emoji by typing the emoji code, pasting Unicode, or using Zulip’s built-in emoji menu.
+
+### Using the emoji menu
+1. Create either a "**New stream message**" or a "**New private message**".
```

Comment:
For this, I'd probably just stay "start composing or editing a message, e.g. by clicking on a message to start a reply".  My reasoning here is that usually the way people start composing messages is by clicking on them to reply.

This feedback also applies to the below text.